content/projects/morse-code/_index.md

```diff
@@ -57,4 +58,22 @@ For a problem like this one, unit tests are more appropriate.
 
 This is optional.
 
-Add unit tests to your code. Leave the assertions in there just so that we can see that you know how to hit the requirements we laid out. But if you have some passing tests as well that would be sweet.
\ No newline at end of file
+Add unit tests to your code. Leave the assertions in there just so that we can see that you know how to hit the requirements we laid out. But if you have some passing tests as well that would be sweet.
+
+## Instructions for Reviewers
+
+- Ensure that both functions work with alphabets, numbers and all types of special characters that exist on a keyboard. 
+- Make sure that both functions returns and not prints the output.
+- When using assertions make sure that the assert statements are checking if the input and output lengths and input and output spaces are the same by checking if they are equal.
+- If Unit tests are used, make sure that there is a test that checks the input and output lengths and ensure that imports and exports was used correctly.
+- Make sure that the assertions do not print an error if empty strings are passed in both functions, e.g 
+
+```
+lettersToMorseCode("") // shouldn't print an assertion failure
+morseCodeToLetters("") // shouldn't print an assertion failure
+```
+
+## Mark as excellent if:
+
+- An assertion function was used to assert if both functions input and output data are the same
```

Comment:
We've been requesting this as a compulsory requirement. Do we now consider this use of an assert function as optional?